backend/dataall/modules/worksheets/api/resolvers.py

```diff
@@ -69,3 +70,49 @@ def run_sql_query(context: Context, source, environmentUri: str = None, workshee
 def delete_worksheet(context, source, worksheetUri: str = None):
     with context.engine.scoped_session() as session:
         return WorksheetService.delete_worksheet(session=session, uri=worksheetUri)
+
+
+@is_feature_enabled('modules.worksheets.features.nlq')
+def text_to_sql(
+    context: Context,
+    source,
+    environmentUri: str = None,
```

Comment:
why are these typing hints setting None as default? Aren't these fields non-nullable in gql?

Reply:
yup I will fix here and for other worksheet resolvers to match how the GQL Schema is defined